mod.ts

```diff
@@ -11,28 +12,39 @@ interface Parser {
 
 type LoadOptions = {
   configPath?: string;
+  env?: string;
 };
 
 interface RuntimeAPI {
-  getEnvVar(key: string): string | undefined;
+  envName: string;
   getEnvVars(): Configs;
-  readFile(path: string): string;
+  getRuntimeEnv(): string | undefined;
+  readFileIfExist(path: string): string | undefined;
 }
 
 class DenoAPI implements RuntimeAPI {
+  envName = "DENO_ENV";
   getEnvVars() {
     return Deno.env.toObject();
   }
-  getEnvVar(key: string) {
-    return Deno.env.get(key);
+  readFileIfExist(path: string) {
+    try {
+      return Deno.readTextFileSync(path);
+    } catch (e) {
+      console.error(e);
+      return undefined;
+    }
   }
-  readFile(path: string) {
-    return Deno.readTextFileSync(path);
+  getRuntimeEnv() {
+    return Deno.env.get("DENO_ENV");
   }
 }
 
 export class Coffee {
-  defaultOptions: LoadOptions = { configPath: "./config" };
+  defaultOptions: LoadOptions = {
```

Comment:
make it private? I'm not with the API that the user can change the `default` config.

Reply:
I don't think so, Usually users can change default configs, and also it breaks the integration tests

Reply:
if so, they are not `default`, they are `configurable options`.
as I mentioned before I'm not with the name.

Reply:
OK